src/solvers/dg_simplices/fluxdiff.jl

```diff
@@ -0,0 +1,182 @@
+"""
+  function hadamard_sum_ATr!(du, ATr, volume_flux, u, skip_index=(i,j)->false)
+
+Computes the flux difference ∑_j Aij * f(u_i, u_j) and accumulates the result into `du`. 
+- `du`, `u` are vectors
+- `ATr` is the transpose of the flux differencing matrix `A`. The transpose is used for 
+faster traversal since matrices are column major in Julia. 
+"""
```

Comment:
```suggestion
"""
    hadamard_sum_ATr!(du, ATr, volume_flux, u, skip_index=(i,j)->false)

Computes the flux difference ∑_j A[i, j] * f(u_i, u_j) and accumulates the result into `du`. 
- `du`, `u` are vectors
- `ATr` is the transpose of the flux differencing matrix `A`. The transpose is used for 
  faster traversal since matrices are column major in Julia. 
"""
```